### DISCUSSION TITLE
I got charged for the free plan

### DISCUSSION THREAD

**Discussion by babat00nday** [community/community#38104] [Copilot Conversations]
Hi,

I wanted to test out the co-pilot without auto subscribing for the service.
Unfortunately, I was charged for it.
Can you please refund the charge.

Regards,
Tunde.

**Comment by david-engelmann** [comment]
[This document has all the info on disputing charges](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#l-cancellation-and-termination)
